src/main/java/seedu/weme/logic/commands/UnstageCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.weme.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.weme.commons.core.Messages;
+import seedu.weme.commons.core.index.Index;
+import seedu.weme.logic.commands.exceptions.CommandException;
+import seedu.weme.model.Model;
+import seedu.weme.model.meme.Meme;
+
+/**
+ * Unstage Command.
+ */
+public class UnstageCommand extends Command {
```

Comment:
MemeUnstageCommand? or will this be usable for Templates as well.

Reply:
This is only for export tab. The index in unstage 2 refers to the index in the exported list.

Reply:
hmm yeah i dont mind, just leave it be then.